integration-tests/enterprise-modscan/docker-compose.yaml

```diff
@@ -0,0 +1,79 @@
+version: '3.7'
+
+services:
+  modscan-server:
+    image: quay.io/hiddenlayer/distro-enterprise-modelscanner:latest
+    container_name: scanner-server
+    ports:
+      - "8000:8000"
+    depends_on:
+      - redis
+      - minio
+    environment:
+      - AWS_ENDPOINT=http://modelscan-s3:9000
+      - AWS_ACCESS_KEY_ID=${MINIO_ROOT_USER}
+      - AWS_SECRET_ACCESS_KEY=${MINIO_ROOT_PASSWORD}
+      - HL_FILE_STORAGE_PROVIDER=AWS:S3
+      - HL_MODEL_SCANNER_ENABLE_RDS_SCANNING=true
+      - HL_MODEL_SCANNER_ENABLE_UNSUPPORTED_FILES_TYPE_AS_UNKNOWN=true
+      - HL_MODEL_SCANNER_REQUIRE_TENANT_ID=false
+      - HL_STORAGE_SCANS=hl-modelscan
+      - REDIS_ENDPOINT=modelscan-redis:6379
+      - REDIS_PASSWORD=${REDIS_PASSWORD}
+ 
+  modscan-worker:
+    image: quay.io/hiddenlayer/distro-enterprise-modelscanner:latest
+    container_name: scanner-worker
+    entrypoint: /modelscan/worker
+    environment:
+      - AWS_ENDPOINT=http://modelscan-s3:9000
+      - AWS_ACCESS_KEY_ID=${MINIO_ROOT_USER}
+      - AWS_SECRET_ACCESS_KEY=${MINIO_ROOT_PASSWORD}
+      - HL_FILE_STORAGE_PROVIDER=AWS:S3
+      - HL_MODEL_SCANNER_ENABLE_RDS_SCANNING=true
+      - HL_MODEL_SCANNER_ENABLE_UNSUPPORTED_FILES_TYPE_AS_UNKNOWN=true
+      - HL_MODEL_SCANNER_REQUIRE_TENANT_ID=false
+      - HL_STORAGE_SCANS=hl-modelscan
+      - REDIS_ENDPOINT=modelscan-redis:6379
+      - REDIS_PASSWORD=${REDIS_PASSWORD}
```

Comment:
It's a shame to have the same 10-line environment block copy-pasted. To eliminate the duplications, ChatGPT says you can:
	1.	Define environment variables once using an anchor (&).
	2.	Reference them using an alias (*) in other services.
Example:

```
x-common-environment: &common-environment
  - VAR1=value1
  - VAR2=value2
  - VAR3=value3
```

then:

```
services:
  service1:
    image: your-image1
    environment:
      <<: *common-environment
  
```
Caveat: I haven't used this YAML feature myself, but looks credible.